schema.sql

```diff
@@ -1,5 +1,10 @@
 /* Database schema to keep the structure of entire database. */
 
-CREATE TABLE animals (
-    name varchar(100)
+CREATE TABLE animals(
+    id int,
+    name varchar(50),
```

Comment:
- [ ] [OPTIONAl] It will be better 😃 if you use the primary key for the column id to make it unique and no duplicate values this is from the best practice 
- [ ] [OPTIONAl] Also consider adding the auto-increment option to the column id this is very helpful to increase the id automatically when you add other records💯🚀
- [ ] [OPTIONAl] Also it will be better if you add the NOT NULL option for the id to not make the value of it null, so it will be better if you follow the best practice and add the option NOT NULL for the id to not make the value of it null💯🚀

- Hint: 

```snippet
  id int NOT NULL IDENTITY(1,1)
  
  `PRIMARY KEY (id)`
```